src/SlashCommands.js

```diff
@@ -282,6 +282,63 @@ var commands = {
             }
         }
         return reject(this.getUsage());
+    }),
+
+    // Verify a user, device, and pubkey tuple
+    verify: new Command("verify", "<userId> <deviceId> <deviceSigningKey>", function(room_id, args) {
+        if (args) {
+            var matches = args.match(/^(\S+) +(\S+) +(\S+)?$/);
+            if (matches) {
+                var userId = matches[1];
+                var deviceId = matches[2];
+                var device = MatrixClientPeg.get().getStoredDevice(userId, deviceId);
+                if (!device) {
+                    return reject("Unknown (user, device) pair: (" + userId + ", " + deviceId + ")");
+                }
+
+                var fingerprint = matches[3];
+
+                if (device.getFingerprint() === fingerprint) {
+
+                    var QuestionDialog = sdk.getComponent("dialogs.QuestionDialog");
+                    return success(Modal.createDialog(QuestionDialog, {
+                        title: "Approve Validated device",
```

Comment:
Ah, sorry, this is a copy-paste induced casing problem.  Will drop the 'Q' down to lowercase.